README.md

```diff
@@ -1,40 +1,185 @@
 [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)
 
-Please, note well: this file and the scaffold were generated from [a
-template](https://github.com/kubewarden/rust-policy-template). Make
-this project yours!
+# High-Risk Service Account Blocker
 
-You can use `cargo generate -g https://github.com/kubewarden/rust-policy-template.git`
-to create your Policy from this template.
+A [service
+accounts](https://kubernetes.io/docs/concepts/security/service-accounts/)
+provides an identity for processes that run in a Pod, and maps to a
+ServiceAccount object. When Pods contact the API server, Pods authenticate as a
+particular ServiceAccount.
 
-# Kubewarden policy high-risk-service-account-policy
+It's important to review the privileges granted to a ServiceAccount to prevent
+a workload from performing unwanted operations.
 
-## Description
+The policy can be configured to define a list of resources and operations that
```

Comment:
I would remove the paragraphs about ServiceAccount at the beginning of the readme, and start directly here with "The policy can be configured (...) when a workload is defined, the policy will use the Kubernetes auth API (...)"